### DISCUSSION TITLE
Compatibility with open search

### DISCUSSION THREAD

**Discussion by leedyche** [Smile-SA/elasticsuite#2534] [Q&A] [answered]
Hi Guys,

With the release of magento 2.4.4, adobe are forcing anyone who upgrades on their cloud platform to move from elastic search to OpenSearch https://opensearch.org/  - which i understand is a fork of elastic search. (from what I understand the reason is licensing)

Reasons aside, does anyone know if elasticsuite will run using opensearch? Does anyone have it running on the adobe platform yet?

Im in the middle of debugging indexer issues with elasticsuite and dont want to discover i might have to move away from it when we upgrade to 2.4.4

thanks

**Comment by romainruaud** [comment, marked as answer]
As stated by the release note :

> 🎉 ElasticSuite 2.10.10 is compatible with Magento 2.4.4 which means :
> 
> PHP 8.1 Compatibility
> OpenSearch Compatibility

This was covered mostly here : https://github.com/Smile-SA/elasticsuite/pull/2516

OpenSearch is (at least for now) strictly identical to Elasticsearch 7, so we only had to fix some parts of the code where we were adapting stuff between ES 6 and ES 7 by taking into account the version indicated by the server. Spoiler : since OpenSearch indicates *1* as the version, we had small adaptation to make, but that should be enough actually.

regards
